scripts/mytop.sh

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/perl -w
 #
-# $Id: mytop,v 1.91 2012/01/18 16:49:12 mgrennan Exp $
+# $Id: mytop,v 1.99-maria3 2015/02/13 02:12:27 jweisbuch Exp $
 
 =pod
 
 =head1 NAME
 
-mytop - display MariaDB server performance info like `top'
+mytop - display MySQL server performance info like `top'
```

Comment:
If i recall correctly (it has been a long time), its because this change had been made on the version shipped with MariaDB while mine is based on the upstream version.

The changes i have made are all compatible with MySQL 5.0 while (if i recall correctly) the version shipped with MariaDB is not compatible with a server that doesnt have INFORMATION_SCHEMA.